CMakeLists.txt

```diff
@@ -157,8 +153,11 @@ if (OE_SGX)
   endif ()
 else () # NOT OE_SGX
   # On non-sgx enclaves are built by default on Unix
-  if (UNIX)
-    set(BUILD_ENCLAVES ON)
+  if (NOT DEFINED BUILD_ENCLAVES)
```

Comment:
I might be wrong, but it appears that earlier
BUILD_ENCLAVES was set to ON by default.
But now it is OFF by default on Windows.
@mingweishih  @nonpolarity  Can you please check if this would create problems?

Reply:
Yes. I turn it off because, in the current version, it seems that in non-OE_SGX cases BUILD_ENCLAVES is set to ON only on Unix systems. Then I follow it.